routers/api/v1/repo/migrate.go

```diff
@@ -124,6 +124,8 @@ func Migrate(ctx *context.APIContext, form api.MigrateRepoOptions) {
 		return
 	}
 
+	log.Info("Mirror Interval Party:", form.MirrorInterval)
+
```

Comment:
Presumably this a debug statement that has sneaked in. Either remove it:

```suggestion
```

or make it more general and drop it to log.Debug